### ISSUE TITLE
Spotify uri paste

### ISSUE THREAD

**Issue by tmos** [raelgc/scudcloud#424]
Hi,

Every time I paste a spotify uri, ScudCloud porpose me to create a new snippet. This is weird, and not wanted behaviour.

For the record, a spotify uri looks like this : spotify:track:7Ijsd2peuxqa43goRQEJvo

Regards,
Tom

**Comment by raelgc** [comment]
Hi @tmos and thanks for raising this issue.

I copied and pasted the same URI and it worked fine:

![spotify](https://cloud.githubusercontent.com/assets/263302/14381127/3d54cf1e-fd5a-11e5-9581-cfde73d5729a.png)

Are you using a different desktop environment or different clipboard manager?

**Comment by tmos** [comment]
Ok so I did some move tests, and actually this is the same kind of issue than #425 : when I ctrl+v, it is very ok, but on typematrix bépo keyboards, you have a shortcut key for pasting, and this is the stroke that launch the creation of a new snippet !

So I guess it can be considered as the same issue ?

**Comment by raelgc** [comment]
Thank you for adding more info, @tmos.

This layout sounds really interesting. 

About this issue: I faced one time only, but I'm not able to reproduce it (while pasting an image, the snippet dialog is displayed after it). 

If we found a way to always reproduce it, then easily it'll be fixed. I'll try a bit.

**Comment by tmos** [comment]
Yeah, this alyout is very interesting, and very helpfull to rest fingers
articulations :)

Do you know any way I could log the signal sent by the keystroke ? Like a
log or a program that could explicit the code sent by the layout ?

2016-04-11 12:28 GMT+02:00 Rael Gugelmin Cunha notifications@github.com:

**Comment by raelgc** [comment]
I think we can add this just for test in ScudCloud. Let me take a look here and I'll get back to you.
